recipes/basemap-data-hires/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+    name: basemap-data-hires
+    version: 1.0.7
+
+source:
+  git_url: https://github.com/matplotlib/basemap.git
+  git_tag: v1.0.7rel
+
+build:
+    number: 0
+
+requirements:
+    build:
+        - basemap
+    run:
+        - basemap
+
+test:
+    imports:
+        - mpl_toolkits.basemap
```

Comment:
Still need to release conda-smithy for that to happen. Only takes 5 mins though. I'll line up some more changes and will cut a release tomorrow unless you want to get this in before then?